src/OrbitClientModel/CaptureDeserializer.cpp

```diff
@@ -178,9 +182,36 @@ ErrorMessageOr<CaptureListener::CaptureOutcome> LoadCaptureInfo(
     frame_track_function_ids.insert(function_id);
   }
 
-  capture_listener->OnCaptureStarted(std::move(process), std::move(instrumented_functions),
-                                     std::move(selected_tracepoints),
-                                     std::move(frame_track_function_ids));
+  CaptureStarted capture_started;
+  capture_started.set_process_id(capture_info.process().pid());
+  capture_started.set_executable_path(process.full_path());
+  capture_started.set_process_id(process.pid());
+  capture_started.set_capture_start_timestamp_ns(0);  // Do we have this information anywhere?
```

Comment:
> Do we have this information anywhere?

I don't remember that we do. I believe just the timestamp of the first event, or the min timestamp.

But I think that such comment shouldn't be left around, rather it should be stated why 0 is used.

Reply:
removed.